docs/ops_heroku_settings.md

```diff
@@ -0,0 +1,36 @@
+# OPS and Heroku Settings
+
+## Staging
+
+Builds to staging are triggered by commits to `master`.
+
+Staging URL is [foundation.mofostaging.net](https://foundation.mofostaging.net)
+
+## Production
+
+Production deployments are done by promoting Staging in the Heroku pipeline.
+
+Production URL is [foundation.mozilla.org](https://foundation.mozilla.org)
+
+## Domain Redirect
+
+Enable domain redirection by setting `DOMAIN_REDIRECT_MIDDLEWARE_ENABLED` to `True`. This will enable a middleware function that checks every request, and return a 307 redirect to the first listed domain in `TARGET_DOMAINS` if the host header does not match one of the domains specified there.
```

Comment:
```suggestion
Enable domain redirection by setting `DOMAIN_REDIRECT_MIDDLEWARE_ENABLED` to `True`. This will enable a middleware function that checks every request, and returns a 307 redirect to the first listed domain in `TARGET_DOMAINS` if the host header does not match one of the domains specified there.
```